RyanNoelk/OpenEats
# Adding default-superagent to set headers in all ajax calls we make

To Test: 
- make sure all the endpoints still work :) 
- Take your time with it since it effects the whole site.
- Make sure to create a few recipes as well.
